packages/gatsby-source-mongodb/src/mapping.js

```diff
@@ -1,4 +1,7 @@
-module.export = function(node, key, text, createNode) {
+const _ = require(`lodash`),
+  crypto = require(`crypto`)
+
+module.exports = function(node, key, text, createNode) {
```

Comment:
Can you tell me why this is needed? How is gatsby using this internally? Just for my understanding :)